kvmagent/kvmagent/plugins/host_plugin.py

```
@@ -2295,8 +2307,7 @@ def change_password(self, req):
     def identify_host(self, req):
         cmd = jsonobject.loads(req[http.REQUEST_BODY])
         rsp = kvmagent.AgentResponse()
-        sc = shell.ShellCmd("ipmitool chassis identify %s" % cmd.interval)
-        sc(True)
+        bash_errorout("ipmitool chassis identify %s" % cmd.interval, timeout=IPMITOOL_TIMEOUT)
```

Ruff (0.15.7)

[error] `bash_errorout` may be undefined, or defined from star imports

(F405)